Circle.cpp

```diff
@@ -1,14 +1,33 @@
 #ifndef Circle_cpp
 #define Circle_cpp
+
+#define _USE_MATH_DEFINES
+
 #include <iostream>
-#include "Draw.cpp"
+#include <math.h>
+
+#include "Shape.cpp"
+#include "Area.cpp"
+#include "Input.cpp"
+
 using namespace std;
-class Circle : public Draw
+class Circle : public Shape , public Input
```

Comment:
`Input` contains functions which serve no purpose in `Circle` i.e. `inputLength` and `inputWidth`.